group/templates/group/groupDetail.html

```diff
@@ -231,21 +244,20 @@ <h4 class="modal-title">Edit Annowence</h4>
     <table class="table table-hover">
       <thead>
         <tr>
-          <th class='col-md-2'>Name</th>
-          <th class='col-md-2'>Adding time</th>
+          <th>Name</th>
+          <th>Adding time</th>
```

Comment:
I think we don't need this column because student's register date is not important for group.